articles/azure/azs-how-use-azure-api.md

```diff
@@ -112,36 +124,54 @@ To obtain an access token:
+    ```
+
+    ### [PowerShell](#tab/tabid-4)
+
+    ```powershell
+    # Add access token to header object for subsequent API requests
+    $AuthHeader = @{"Authorization" = "Bearer $($AuthResp.access_token)"}
+
+    # Query the Azure Stack API for subscriptions
+    Invoke-RestMethod -Method "GET" -Uri "https://management.frn00006.azure.ukcloud.com/subscriptions" -Headers $AuthHeader -Body @{"api-version" = "2016-05-01"} -ContentType "application/x-www-form-urlencoded"
     ```
 
+    ***
+
 ## Calling Azure Stack API endpoints
 
 A REST request URI consists of:
 
-`<URI-scheme>://<URI-host>/<resource-path>?<query-string>`
+   `<URI-scheme>://<URI-host>/<resource-path>?<query-string>`
 
 where:
 
-- `URI-scheme` is the protocol used to transmit the request, for example `http` or `https`.
+- **URI-scheme** is the protocol used to transmit the request. For example `http` or `https`.
+
+- **URI-host** is the domain name or IP address of the server where the REST service endpoint is hosted. For example: `management.frn00006.azure.ukcloud.com`.
 
-- `URI-host` is the domain name or IP address of the server where the REST service endpoint is hosted, for example `management.local.azurestack.external`.
+- **resource-path** is the resource or resource collection, which may include multiple segments used by the service in determining the selection of those resources. For example: `beta/applications/00003f25-7e1f-4278-9488-efc7bac53c4a/owners` is the resource path to a specific application's owners within the applications collection.
 
-- `resource-path` is the resource or resource collection, which may include multiple segments, used by the service in determining the selection of those resources. For example `beta/applications/00003f25-7e1f-4278-9488-efc7bac53c4a/owners` is the resource path to a specific application's owners within the applications collection.
+- **query-string** provides additional parameters, such as the API version or resource selection criteria.
 
-- `query-string` provides additional simple parameters, such as the API version or resource selection criteria.
+    >[!NOTE]
+    > For Bash, you can add the **query-string** to the end of the request URI following a question mark. For example, to specify use of a specific API version: `https://management.frn00006.azure.ukcloud.com/subscriptions?api-version=2016-05-01`. <br>For PowerShell, you can provide a **query-string** in the **-Body** parameter hash table. For example: `-Body @{"api-version" = "2016-05-01"}`.
 
 The syntax of an Azure Stack request URI is:
 
-`https://management.local.azurestack.external/<subscription-id>/resourcegroups/<resource-group>/providers/<provider>/<resource-path>?<filter-expression>&api-version=<api-version>`
+`https://management.frn00006.azure.ukcloud.com/<subscription-id>/resourcegroups/<resource-group>/providers/<provider>/<resource-path>?<filter-expression>&api-version=<api-version>`
```

Comment:
It would be easier to follow the example if api-version came before the filter expression